spec/codelog/command/step/version_spec.rb

```diff
@@ -120,4 +120,17 @@
       described_class.run '1.2.3', '2012-12-12'
     end
   end
+
+
+  describe '#changes_hash' do
+    context "when a not parseable yml file is given" do
+      subject {described_class.new('1.2.3', '2012-12-12')}
+
+      it 'aborts with an appropriate error message' do
+        double_error = double("Psych::SyntaxError", problem: "found character that cannot start any token", line: "2", file: "spec/fixtures/files/not_parseable.yml")
+        allow(Dir).to receive(:"[]") { ["spec/fixtures/files/not_parseable.yml"] }
+        lambda { subject.run }.should raise_error SystemExit
+      end
```

Comment:
It would also be good to verify if the error message is being displayed correctly

Reply:
@emilianoLeite I added the suggested message verification.